build.gradle

```
@@ -23,7 +23,7 @@ if (env.BUILD_NUMBER) {
 group = 'com.direwolf20.buildinggadgets'
 archivesBaseName = 'buildinggadgets'
 
-String forgeVersion = "$minecraft_version-${forge_major}.$forge_minor"
+String forgeVersion = "$minecraft_version-${forge_major}.${forge_minor}"
```

If you're adding them for the last one then add them for the first one too XD.
```suggestion
String forgeVersion = "${minecraft_version}-${forge_major}.${forge_minor}"
```